#basically a list of gestures as noted in config file

int32 CMD_GESTURE_START       = 2

int32 cmd
float64[] param  
